ntoskrnl/mm/ARM3/section.c

```diff
@@ -9,6 +9,7 @@
 /* INCLUDES *******************************************************************/
 
 #include <ntoskrnl.h>
+#include <ntintsafe.h>
```

Comment:
I think it's time to add this to ntoskrnl.h. I can do that in my upcoming commit though if you don't want to.